test/python/test_nexus_multipath.py

```diff
@@ -336,3 +409,71 @@ def test_nexus_multipath_add_3rd_path(
 
     # wait for fio to complete
     time.sleep(15)
+
+
+@pytest.mark.timeout(60)
+def test_nexus_multipath_remove_3rd_path(
+    create_nexus_dev,
+    create_nexus_2_no_destroy,
+    connect_nexus_2,
+    create_nexus_3_dev,
+    start_fio,
+    destroy_nexus_2,
+):
+    """Create 3 nexuses, connect over NVMe, start fio, destroy 2nd nexus."""
+
+    dev = create_nexus_dev
+    dev2 = connect_nexus_2
+    start_fio
```

Comment:
start_fio() fixture doesn't yield any value, but None. What's the point of using the yielded None here ?

Reply:
I've removed this line, and a similar one in the next test, as it's already executed as a fixture and there is no return value anyway.